### PR TITLE
Fixes/#722 marginal costs conventional

### LINKED ISSUE #722: Conventional power plants are missing marginal costs in 'eGon2035'
Conventional power plants are missing marginal costs in 'eGon2035'. Data need to be added to 'scenario_parameter' and assigned to corresponding plants.

### PR DESCRIPTION
**Pull request by IlkaCu**
Fixes #722  . 

## Before merging into `dev`-branch, please make sure that

- [x] the `CHANGELOG.rst` was updated.
- [x] new and adjusted code is formated using `black` and `isort`.
- [x] the `Dataset`-version is updated when existing datasets are adjusted. 
- [x] the branch was merged into the `continuous-integration/run-everything-over-the-weekend-v2`- branch.
- [x] the workflow is running successful in `test mode`.
- [x] the workflow is running successful in `Everything` mode.

### CODE REVIEW

**Comment by IlkaCu** [comment]
Marginal costs are still missing for foreign generators

File: src/egon/data/datasets/chp_etrago.py [hunk 1]
```diff
@@ -107,6 +108,12 @@ def insert():
     chp_el["link_id"] = range(
         db.next_etrago_id("link"), len(chp_el) + db.next_etrago_id("link")
     )
+
+    # Add marginal cost
+    chp_el["marginal_cost"] = get_sector_parameters("gas", "eGon2035")[
+        "marginal_cost"
+    ]["CH4"]
+
```
Comment (ClaraBuettner) [inline src/egon/data/datasets/chp_etrago.py:116]: The fuel costs for CH4 are already added to the CH4-feedin generators. When you add these costs here, they are assigned twice.

File: src/egon/data/datasets/chp_etrago.py [hunk 2]
```diff
@@ -227,12 +239,18 @@ def insert():
         db.next_etrago_id("link"), len(chp_el_ind) + db.next_etrago_id("link")
     )
 
+    # Add marginal cost
+    chp_el_ind["marginal_cost"] = get_sector_parameters("gas", "eGon2035")[
+        "marginal_cost"
+    ]["CH4"]
```
Comment (ClaraBuettner) [inline src/egon/data/datasets/chp_etrago.py:245]: see above

File: src/egon/data/datasets/chp_etrago.py (hunk 1 above)
Comment (IlkaCu) [reply src/egon/data/datasets/chp_etrago.py:116]: I changed it to only consider VOM for gas CHPs

**Review by ClaraBuettner — APPROVED** [review]

**Pull request merged**